ecs-init/docker/docker_test.go

```diff
@@ -639,3 +639,29 @@ func TestStopAgent(t *testing.T) {
 		t.Error("Error should not be returned")
 	}
 }
+
+func TestContainerLabels(t *testing.T) {
```

Comment:
Thanks for adding these unit tests. could you please add a case for empty / no labels?